contract/scripts/run-chain.sh

```diff
@@ -11,4 +11,4 @@ waitForBlock 1
 make -C /workspace/contract mint100
 
 # bring back chain process to foreground
-wait
\ No newline at end of file
+wait
```

Comment:
This is a fixup (a change to some earlier change in this PR), not a fix (a change in the behavior of the code w.r.t. what was previously released or merged). In agoric-sdk, we auto-squash on merge, but I don't think that's set up here, so please squash and force-push.

Not critical, but let's work toward [Tidy Commit History](https://github.com/Agoric/agoric-sdk/wiki/Tidy-Commit-History) in general.